chat-models/google-vertexai.md

```diff
@@ -13,7 +13,7 @@
 
 <figure><img src="../.gitbook/assets/gcp_credential/vertex_AI_enable.png" alt="" width="563"><figcaption></figcaption></figure>
 
-## Create credential file (Optional)
+## Create credential file _(Optional)_
```

Comment:
why the change of this?

Reply:
standardize the way of writing [OPTIONAL] and (Optional) to become _(Optional)_